### PR TITLE
Modernize tracking instrumentation events

### PR DESCRIPTION
**Pull request by bormind**
# What

* Renamed tracking events related to the Facebook friends following.
* Implemented "Loaded More Friends" event tracking. 

# Why

As a part of an effort of renaming (modernizing) tracking events and adding missing once.

# How

* Deprecated old event names and added new ones.
* Implemented "Loaded More Friends" event tracking. This event indicates that more then one page of friends was requested. As part of this event two properties are passed: "source" and "page_count"
* Fixed tests to accommodate deprecated and new event names and properties
* Added test for  "Loaded More Friends" tracking functionality

# See 👀

Trello: [Modernize following instrumentation](https://trello.com/c/0sqaG7c7/471-modernize-following-instrumentation)

# TODO

We probably have to come up with a new name for "Facebook Friend Decline Follow All" event and deprecate the current one

### CODE REVIEW

File: Library/Koala/Koala.swift [hunk 1]
```diff
@@ -1300,31 +1300,57 @@ public final class Koala {
   }
 
   public func trackDeclineFriendFollowAll(source: FriendsSource) {
+    // deprecated
+//    self.track(event: "Facebook Friend Decline Follow All", properties: deprecatedProps)
```
Comment (justinswart) [inline Library/Koala/Koala.swift]: Let's wait until we have the name before merging 👍

File: Library/ViewModels/FindFriendsViewModelTests.swift [hunk 2]
```diff
@@ -260,6 +272,41 @@ final class FindFriendsViewModelTests: TestCase {
...
+
+      XCTAssertEqual(
+        ["Find Friends View", "Viewed Find Friends", "Loaded More Friends"], self.trackingClient.events)
+
+      XCTAssertEqual([nil, "activity", "activity"],
+                     self.trackingClient.properties.map { $0["source"] as! String? })
+
+      XCTAssertEqual([nil, nil, 2],
+        self.trackingClient.properties.map { $0["page_count"] as! Int? })
+
```
Comment (justinswart) [inline Library/ViewModels/FindFriendsViewModelTests.swift]: Can delete this new line ☝️

**Comment by justinswart** [comment]
@theginbin ah of course re `source` attribute, thanks!

**Comment by bormind** [comment]
@theginbin, @justinswart thank you for the review.

**Comment by theginbin** [comment]
merge away @bormind!

**Comment by bormind** [comment]
Great thanks @theginbin, @justinswart for helping me through!

**Pull request merged**